Dockerfile-default-cpu

```diff
@@ -10,12 +10,16 @@ ARG TENSORFLOW_PIP
 ARG TORCH_PIP
 ARG TORCHVISION_PIP
 RUN if [ "$TENSORFLOW_PIP" ]; then \
+		export HOROVOD_WITH_TENSORFLOW=1 && \
         if [ "$TARGETPLATFORM" = "linux/arm64" ]; then \
             TENSORFLOW_AARCH64_PIP=$(printf '%s' "$TENSORFLOW_PIP" | sed 's/cpu/aarch64/') \
             && pip install $TENSORFLOW_AARCH64_PIP; \
         else \
             pip install $TENSORFLOW_PIP; \
         fi; \
+	else \
```

Comment:
I don't think this complex dual variable WITH & WITHOUT logic is necessary.

Looking at the [makefile](https://github.com/horovod/horovod/blob/d15c9147debe3af9929a2c283b5e6d5aefa0cad7/horovod/tensorflow/CMakeLists.txt#L9), if `HOROVOD_WITH_TENSORFLOW=0` and there's no tensorflow present in the image, it'd just build without it.

i.e. we can just have `HOROVOD_WITH_TENSORFLOW` as one ARG.

Reply:
Ah, this is true. The [horovod documentation](https://horovod.readthedocs.io/en/stable/install_include.html) was a little confusing. They recommend using WITH = 1 or WITHOUT = 1, but I think using one ARG is definitely cleaner.